### PR TITLE
update packages

### PR DESCRIPTION
**Pull request by angelofb**
updated all packages except for svelto, didn't found on docs how to update to latest version

### CODE REVIEW

**Comment by friflo** [comment]
Interesting - you added a .NET wrapper for Flecs - the ECS pioneer.
Curious for the perf resuts.

I saw a small glitch in your commit - will add a note.

File: source/Ecs.CSharp.Benchmark/CreateEntityWithOneComponent/FlecsNet.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+using Flecs.NET.Core;
+
+namespace Ecs.CSharp.Benchmark
+{
+    public partial class CreateEntityWithOneComponent
+    {
+        [Context]
+        private readonly FlecsNetBaseContext _flecs;
+
+        [BenchmarkCategory(Categories.Arch)]
```
Comment (friflo) [inline source/Ecs.CSharp.Benchmark/CreateEntityWithOneComponent/FlecsNet.cs]: Should use its own string id - not Arch

File: source/Ecs.CSharp.Benchmark/CreateEntityWithThreeComponents/FlecsNet.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+using Flecs.NET.Core;
+
+namespace Ecs.CSharp.Benchmark
+{
+    public partial class CreateEntityWithThreeComponents
+    {
+        [Context]
+        private readonly FlecsNetBaseContext _flecs;
+
+        [BenchmarkCategory(Categories.Arch)]
```
Comment (friflo) [inline source/Ecs.CSharp.Benchmark/CreateEntityWithThreeComponents/FlecsNet.cs]: Should use its own string id - not Arch

File: source/Ecs.CSharp.Benchmark/Contexts/FlecsNetBaseContext.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            public int Value;
+        }
+
+        internal struct Component3
+        {
+            public int Value;
+        }
+    }
+
+    internal class FlecsNetBaseContext : IDisposable
```
Comment (Doraku) [inline source/Ecs.CSharp.Benchmark/Contexts/FlecsNetBaseContext.cs]:
```suggestion
    internal sealed class FlecsNetBaseContext : IDisposable
```

File: source/Ecs.CSharp.Benchmark/Contexts/FlecsNetBaseContext.cs [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    internal sealed class FlecsNetBaseContext : IDisposable
+    {
+        public World World { get; }
+
+        public FlecsNetBaseContext()
+        {
+            World = World.Create();
+        }
+
+        public virtual void Dispose()
```
Comment (Doraku) [inline source/Ecs.CSharp.Benchmark/Contexts/FlecsNetBaseContext.cs]:
```suggestion
        public void Dispose()
```

**Pull request merged**